analyzers/rspec/cs/S5693_c#.html

```diff
@@ -59,6 +76,23 @@ <h2>Compliant Solution</h2>
     }
 }
 </pre>
+<p>In <a href="https://docs.microsoft.com/en-us/troubleshoot/aspnet/create-web-config">Web.config</a>:</p>
```

Comment:
You should check the other rules with web.config, but normally we put links at the end, all together

Reply:
Also, I don't think the troubleshoot link is the best to share

Reply:
TBH the way it was done in the RSPEC I like more, since those links at the end concern only security related links. And all non-security related appear in the text where they need to be.

Updated PR, please give your feedback.

Reply:
I agree it's debatable which is best. I favor consistency however until a cross-LT guideline is created.